specifyweb/frontend/js_src/lib/components/Forms/DeleteBlocked.tsx

```diff
@@ -159,6 +159,7 @@ function BlockerPreview({
           headerButtons={undefined}
           ids={resolvedIds}
           isDependent={false}
+          totalCount={resolvedIds.length}
```

Comment:
this line probably be removed as that component will already get total count from resolvedIds.length